.github/workflows/_buildpacks-release.yml

```diff
@@ -0,0 +1,245 @@
+name: _buildpacks-release
+
+on:
+  workflow_call:
+    inputs:
+      app_id:
+        description: Application ID of GitHub application (Linguist)
+        type: string
+        required: true
+      dry_run:
+        required: false
+        type: boolean
+        default: false
+        description: Flag used for testing purposes that prevents actual changes to the heroku/builder repo
+      ip_allowlisted_image:
+        description: The image to use to run jobs that require IP allowlist privileges
+        type: string
+        required: false
+        default: pub-hk-ubuntu-22.04-small
+    secrets:
+      app_private_key:
+        description: Private key of GitHub application (Linguist)
+        required: true
+      cnb_registry_token:
+        required: true
+        description: The token to login to the CNB registry with
+      docker_hub_user:
+        required: true
+        description: The username to login to Docker Hub with
+      docker_hub_token:
+        required: true
+        description: The token to login to Docker Hub with
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  compile:
+    name: Compile Buildpacks
+    runs-on: ubuntu-latest
+    outputs:
+      buildpacks: ${{ steps.generate-buildpack-matrix.outputs.buildpacks }}
+      version: ${{ steps.generate-buildpack-matrix.outputs.version }}
+      changelog: ${{ steps.generate-changelog.outputs.changelog }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Install musl-tools
+        run: sudo apt-get install musl-tools --no-install-recommends
+
+      - name: Update Rust toolchain
+        run: rustup update
+
+      - name: Install Rust linux-musl target
+        run: rustup target add x86_64-unknown-linux-musl
+
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v2.5.1
+
+      - name: Install libcnb-cargo
+        run: cargo install libcnb-cargo
+
+      - name: Package buildpacks
+        id: libcnb-package
+        run: cargo libcnb package --release
+
+      - name: Generate buildpack matrix
+        id: generate-buildpack-matrix
+        uses: heroku/languages-github-actions/.github/actions/generate-buildpack-matrix@main
+
+      - name: Generate changelog
+        id: generate-changelog
+        uses: heroku/languages-github-actions/.github/actions/generate-changelog@main
+        with:
+          version: ${{ steps.generate-buildpack-matrix.outputs.version }}
+
+      - name: Cache buildpacks
+        uses: actions/cache/save@v3
+        with:
+          key: ${{ github.run_id }}-compiled-buildpacks
+          path: target/buildpack
```

Comment:
I wonder if the `generate_buildpack_matrix` command should output this value, and then it could be accessed via eg:
`${{ steps.generate-buildpack-matrix.outputs.output_dir }}`
...or similar?

That way all the Rust/libcnb related paths are at least in one place - given there is already the path `target/` hardcoded in `src/commands/generate_buildpack_matrix/command.rs` here:
https://github.com/heroku/languages-github-actions/pull/91/files#diff-7abed701849efef91f6445231b0efc6baa56c948c52a602b1028e4aeb554e4ccR81

Longer term, maybe `cargo libcnb package` should be made to accept an `--output-dir` path, which would mean the workflows in this repo would be in full control of the paths in question, so don't need to hardcode paths?

Reply:
i like the longer term idea that we control that output path through the whole process.  in that case i think we'd define it as an optional input to the workflow.  

for now, i'm not happy to have hardcoded aspects but i think this is fine for now.  i'll create a new work item that outlines what changes to `libcnb-cargo` and this repo would be needed to coordinate this.